src/content/development/contribution-guide/_index.en.md

```diff
@@ -58,7 +58,35 @@ To be expanded
 
 #### Pre-Planning
 
-To be expanded
+The week before the release is dedicated to planning the next release.
+Planning accounts for [tasks](#tasks) and [bugs](#bugs) which we intend to solve in the next version.
+Planning meetings will be held, focusing on the [Backlog board].
+
+For a [task](#tasks) to be eligible for the next version, it needs to fulfill these requirements:
+
+* Be part of the [Backlog board].
+* Have a `next-version-candidate` label.
+* Have a description detailing **what** is the issue and optionally **how** is it going to be solved.
+* Have an apropriate sizing label, according to amount of work expected for a single person to completely deliver the task:
+  * *Small*: Work is contained in one sprint and is expected to take less than half the sprint.
+  * *Medium*: Work is contained in one sprint and is expected to take most of the sprint.
+  * *Large*: Work is contained within a release (two-three sprints).
+  * *Extra-Large* = Work can't be contained within a release and would span multiple releases.
+* Any *Large* or *Extra-Large* task must be converted to an [epic](#epics).
+* In case of an [epic](#epics):
+  * Be an issue in the [enhancements] project.
+  * Adhere to the [epic template].
+  * Have a high level break down of the **expected** work, corresponding to the "Definition of Done".
+
+The project team will hold planning meetings, led by the project's "scrum master".
```

Comment:
```suggestion
The project team will hold planning meetings, led by the project's "scrum lead".
```

Reply:
This is the adjective form..
> adjective:
> 
>  master
> 
>     1.
>     having or showing very great skill or proficiency.

It is also how it's referred to on scrum.org - https://www.scrum.org/resources/what-is-a-scrum-master

Reply:
@mkolesnik I know the term, but have you seen https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language?

Reply:
@nyechiel yes i have and this is referring to the noun form of the word not the adjective, unless "masterchef" is also getting rebranded as "mainchef"?

Reply:
I don’t understand how this can be considered an adjective; “scrum master” is a noun group with two nouns, there’s no adjective there.